src/modules/assignment-5/questions/question7/Task7.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import axios from "axios";
+
+const Task7 = () => {
+  const textareaStyle = {
+    height: 400,
+    width: 700,
+    border: "1px solid #ccc",
+    padding: "10px",
+    fontSize: "16px",
+    fontFamily: "monospace",
+    whiteSpace: "pre-wrap",
+    backgroundColor: "#f4f4f4",
+    color: "#333",
+    borderRadius: "5px",
+  };
+
+  const [options, setOptions] = useState({
+    method: "POST",
+    url: "https://onecompiler-apis.p.rapidapi.com/api/v1/run",
+    headers: {
+      "content-type": "application/json",
+      "X-RapidAPI-Key": "a751c5151amshb89a31e7e7bdb07p141368jsnb8f10878d806",
+      "X-RapidAPI-Host": "onecompiler-apis.p.rapidapi.com",
+    },
+    data: {
+      language: "python",
+      files: [
+        {
+          name: "index.py",
+          content: "",
+        },
+      ],
+    },
+  });
+
+  const [input, setInput] = useState("");
+  const [output, setOutput] = useState("");
+
+  const fetchData = async () => {
+    if (input === 0) {
+      console.log("No code found!");
+      return;
+    }
+
+    options.data.files[0].content = input;
+
+    setOptions({ ...options });
+    console.log(options);
+
+    try {
+      const response = await axios.request(options);
+      console.log(response.data);
+
+      if (response.status === "failed") {
+        throw new Error(response.error);
+      }
+
+      if (response.data.stdout === null) setOutput("No response found!");
+      else setOutput(response.data.stdout);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
```

Comment:
![image](https://github.com/ayush1-successive/react-training/assets/83696224/7390788f-0483-480d-9744-3d325ffa255d)
Why this placeholder consist pyhton